domain/service/servicedetails.go

```diff
@@ -29,6 +29,7 @@ type ServiceDetails struct {
 	Description     string
 	PoolID          string
 	ParentServiceID string
+	Parent          *ServiceDetails `json:",omitempty"`
```

Comment:
Since the property is only used by one of the endpoints, it might be better to have another struct with the property (like ServiceDetailsResponse or anonymous in the web layer) and embed the ServiceDetails.